packages/genesys-spark-components/src/components/stable/gux-tag/gux-tag.tsx

```diff
@@ -83,9 +83,15 @@ export class GuxTag {
       <gux-tooltip-title>
         <span>
           <slot
+            name="content"
             aria-hidden="true"
             onSlotchange={this.onSlotChange.bind(this)}
-          />
+          >
+            <slot
+              aria-hidden="true"
+              onSlotchange={this.onSlotChange.bind(this)}
+            />
+          </slot>
```

Comment:
If we are going ahead with this change should we add this to the `v5` epic to be removed?